### PR TITLE
Cap 80 login screen styling

### PR DESCRIPTION
**Pull request by jmkraut**
Did considerable tidy up on the login screen and added in error alert dialogs.
Changed app icon and splash screen.

image
image

### CODE REVIEW

File: lib/auth/login.dart [hunk 1]
```diff
@@ -25,61 +29,119 @@ class LoginPage extends StatefulWidget {
...
+
+  _updateCertVal(bool value) {
+    setState(() {
+      _ignoreCert = !_ignoreCert;
+    });
+
+    userPrefs.setBool('certval', value);
+  }
+
+  Future<void> connect(String username, String password) async {
```
Comment (kressnick25) [inline lib/auth/login.dart]: This does nothing

**Review by kressnick25 — CHANGES_REQUESTED** [review]
Just need to change connect back to original UdContext.connect() rather than wrapped with a Future as this does not do anything currently

File: lib/auth/login.dart [hunk 2]
```diff
@@ -25,61 +29,119 @@ class LoginPage extends StatefulWidget {
...
     // bind local state to udConfig state
     UdConfig.ignoreCertificateVerification(_ignoreCert);
     try {
-      widget.udContext.connect(user.username, user.password);
-      setState(() {
-        _errMessage = null;
-        _isLoading = false;
-      });
+      // widget.udContext.connect(user.username, user.password);
+      await connect(user.username, user.password).then((value) => setState(() {
```
Comment (kressnick25) [inline lib/auth/login.dart]: Change this back to old call

**Review by kressnick25 — CHANGES_REQUESTED** [review]

File: lib/auth/login.dart [hunk 3]
```diff
@@ -88,44 +150,156 @@ class _LoginPageState extends State<LoginPage> {
       DeviceOrientation.portraitDown,
     ]);
     return Scaffold(
-      appBar: AppBar(
-        title: const Text('Login'),
-      ),
+      backgroundColor: Theme.of(context).backgroundColor,
       body: Form(
         key: _formKey,
         child: Column(
```
Comment (yuk0ga) [inline lib/auth/login.dart]: Change this child to a SingleChildScrollView(), and make this Column() the child of it. 
https://www.youtube.com/watch?v=2E9iZgg5TOY
This makes the login screen scrollable, which solves the bottom overflow.
Like this:
Screen Shot 2020-09-05 at 15 51 07

**Review by yuk0ga — CHANGES_REQUESTED** [review]

**Pull request merged**